webapp/packages/plugin-authentication-administration/src/Administration/Users/UserForm/Info/UserFormInfoPart.ts

```diff
@@ -96,6 +96,7 @@ export class UserFormInfoPart extends FormPart<IUserFormInfoState, IUserFormStat
         authRole: getTransformedAuthRole(this.state.authRole),
         enabled: this.state.enabled,
       });
+      this.state.userId = user.userId;
```

Comment:
please add comment that userId is modified by backend and may not match value we sent, so we need to update state